pkg/reconcile/pipeline/infinispan/handler/configure/tls.go

```diff
@@ -48,8 +49,18 @@ func Keystore(i *ispnv1.Infinispan, ctx pipeline.Context) {
 		} else if isUserProvidedPrivateKey() {
 			keystore.Path = consts.ServerOperatorSecurity + "/" + EncryptPemKeystoreName
 			keystore.PemFile = append(keystoreSecret.Data["tls.key"], keystoreSecret.Data["tls.crt"]...)
+		} else {
+			errMsg := fmt.Sprintf("Failed to setup TLS keystore from secret %s", keystoreSecret.Name)
+			_ = ctx.UpdateInfinispan(func() {
+				i.SetCondition(ispnv1.ConditionTLSSecretValid, metav1.ConditionFalse, errMsg)
+			})
+			ctx.Requeue(fmt.Errorf(errMsg))
```

Comment:
I don't think we need to return the `errMsg` here as this isn't an Operator error and we're conveying the message to the user via the Infinispan condition. So `ctx.Requeue(nil)` should be sufficient to avoid excessive error logging.